File: Python/wejscie-wyjscie.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
#
#  szablon.py


def main(args):
    #a = 0 # inicjacja zmiennnej
    a = int(input('Podaj 1. liczbę: '))
    print(a)
    b = int(input('Podaj 2. liczbę: '))
    print(b)
    
    print("Suma: ", a + b)
    print("Różnica: ", a - b)
    print("Iloczyn: ", a * b)
    print("Iloraz: ", a / b)
    
    return 0
    
# duck typing

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: cpp/wejscie-wyjscie.cpp
/*
 * szablon.cpp
 */


#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
	float a, b; //deklaracja zmiennej
    a = b = 0; //inicjacja zmiennej
    
    cout << "Podaj 1. liczbę: ";
    cin >> a;
    cout << a << endl;
    
    cout << "Podaj 2. liczbę: ";
    cin >> b;
    cout << b << endl;
    
    cout << "Suma: " << a + b << endl;
    cout << "Różnica: " << a - b << endl;
    cout << "Iloczyn: " << a * b << endl;
    cout << "Iloraz: " << a / b << endl;

    
    
	return 0;
}

